week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -39,6 +59,13 @@ Write a function that:
 
 function formatPercentage(arr) {
   
+  for (let i=0; i < arr.length; i++){
+    if (arr[i] > 100){arr[i] = 100}
+
+    arr[i] = Math.ceil(arr[i]*100)/100
+    arr[i] = arr[i].toString()
+    arr[i] = arr[i].concat("%")
```

Comment:
Note that you are missing `}` here. I see that you've added one at the end of the file, but that makes the tests to not run. Try placing the brace here and re-running.

Another formatting point, use `;` at the end of the statements. It makes the code less prone to errors and easier to read.